spec/rails_applications/7.1.3/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+v+qpJS9dHtYdjpHPCRVcjajq+Tlb6ONmeul+eI9CFyFtJQMcejkgV8w8H0w90kwselljjrjD+YxZUq8zarwhrmWC+03M4JOwJGDE5C/Iaofa2pT5qtjZSP7/wL9v4G6/lFpKBvT+teCRhXs+6uvcWACyFotEKZq89v6ApMBgzF94sioz5LmY51gFSIk1EhaMuEut0lUWOYn/mzBRA/sdYymEM4ynOg5AglQ2Q1c34JIQtrWnGrtwMD9C/XnQ96vUZcQd4wUQUjYtF0xnJNx/MknaHMdbrS26zXMpmBveh/O8a67x/ozO/c8hjnYeMPMZ0lOdHnbyWsrdpCjhHkKA7foc3KuCkMEsHXvFbWnlip7doOaF4tKwTq9z79gxshYJTzQ2CXrIgDignUcNlecVj3Krkmpb--A0mHiuQO+RWqb84v--VvD6hXbRbSQZ5vYc2YVqjw==
```

Comment:
Do we want to be committing this? Seems sensitive

Reply:
Encrypted: https://edgeguides.rubyonrails.org/security.html#custom-credentials

Inside there it's just a random `secret_key_base` and a few comments